services/ui-src/src/components/reports/ReportProvider.tsx

```diff
@@ -93,6 +93,7 @@ export const ReportProvider = ({ children }: Props) => {
       return result;
     } catch {
       setError(reportErrors.GET_REPORT_FAILED);
+      return;
```

Comment:
extreme nit, we don't need this return here.  Right?

Reply:
@JonHolman without that `return` you'll see a typescript error `Not all code paths return a value.`

Reply:
Thank you, @britt-mo